### ISSUE TITLE
[FEATURE] Highlight seperate cooldown when should be used

### ISSUE THREAD

**Issue by SaschaDaSilva** [Hekili/hekili#2979]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/enhancement) for this request and was not able to find one.
- [X] I edited the title of this feature request (above) so that it describes the issue I am reporting.

### Feature Request

One big "issue" i have with cooldowns or rather a Feature i always wanted in Hekili is that when i activate seperate cooldowns hekili doesnt recommend when to use them.

So basically i have my Rotation in the middle, and my cooldowns right above it, that the addon highlights the cooldown when it should be used. 

Otherwise i have to deactivate the seperate cooldown window, but then my rotation always recommends a cooldown and assumes im gonna use it, therefore recommends different rotation ahead of time when i dont want to use it (for example next pack needs offensive cds)

Can you somehow include a feature like that?

### Contact Information

saschadasilva

**Comment by Hekili** [comment]
/hekili > Toggles > Cooldowns > Show Separately.

**Comment by SaschaDaSilva** [comment]
![Bild_2023-11-20_011613110](https://github.com/Hekili/hekili/assets/68031789/56c3fa72-4bfe-42fe-9718-6c90ba421dcb)

Problem is, when activated seperately, the sequences make no sense at all

The proper sequence (shown when seperate cooldown is toggled off) would be

- Barbed shot > Barbed shot > call of the wild

**Comment by Hekili** [comment]
No; this is the downside of separating CDs into a separate display.  It's not a tray that lights up.  It's showing what's next that has its criteria met.  Barbed Shot is probably going to come after you summon your pet and get Kill Command on CD.
